en/console-and-shells/completion-shell.rst

```diff
@@ -94,20 +106,50 @@ The following bash example comes from the original author::
             opts=$(${cake} Completion subcommands $prev)
             COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
             if [[ $COMPREPLY = "" ]] ; then
-                COMPREPLY=( $(compgen -df -- ${cur}) )
+                _filedir
```

Comment:
@HavokInspiration bash_completion is not provided by default on OS X, but, as you suggested, it can be installed with `brew install bash-completion`. Then by default, the `cake` file should be created in **/usr/local/etc/bash_completion.d/**.

Works perfectly with it:
![autocompletion cake](https://cloud.githubusercontent.com/assets/1652972/12149007/c3ee4458-b4a1-11e5-8157-8df0673e5d8c.png)